### DISCUSSION TITLE
【提问】如何吸引新人进入论坛？

### DISCUSSION THREAD

**Discussion by xgdyp** [X-lab2017/open-wonderland#38] [2️⃣ 聊项目]
论坛里目前发言的同学还比较少，如何能让更多外界的同学参与进来呢？

**Comment by will-ww** [comment]
> 论坛里目前发言的同学还比较少，如何能让更多外界的同学参与进来呢？

你这是个好问题，可以一起来讨论：

1、需要对论坛做些整体规划和运营
2、需要有核心优质的内容输出，才能吸引人
3、首先解决我们自己的问题，要和我们的同学高度相关的事物
4、问题不大，我们肯定有很多好主意

大家来出主意了~ :-)
